### DISCUSSION TITLE
Data Inheritance and Workflow State

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16942] [Q&A / Support] [answered]
Can the [field used to track workflow state](https://pimcore.com/docs/platform/Pimcore/Workflow_Management/Marking_Stores#single_state) for an Object have data inheritance turned on? In this case the idea would be that changing the state on the parent would change the state on all children (assuming they have not overridden the the state field).

**Comment by dpfaffenbauer** [comment]
You can do that programmatically. One workflow triggers the child's for example

**Reply by cancan101** [reply]
Will data inheritance _not_ work though? i.e. if it is set up will it cause both Objects to change states?

**Reply by markus-moser** [reply, marked as answer]
The state should change also for children (but for sure you would not have a workflow transition note in the notes/events list). The question is then if you would also like to allow transitions on the children objects. So it dependes on the exact implementation and use case how it could work, but the state/place information should inherit to the children in your described constellation.
